frontend/dockerfile/shell/lex_test.go

```diff
@@ -44,8 +42,7 @@ func TestShellParserMandatoryEnvVars(t *testing.T) {
 	require.Equal(t, "", newWord)
```

Comment:
This could be simplified to `require.Empty`

Reply:
Ah! Yes, let me update that as well. I'm not a frequent user of testify, so didn't go look for those. My eye landed on the string matching for errors, which felt "incorrect", so I assumed testify had a function for that (it did), which also produced better output on failures, so I updated them 👍